### ISSUE TITLE
249: Converter must be able to work across scenes

### ISSUE THREAD

**Issue by tngreene** [der-On/XPlane2Blender#419]
# Problem
Right now the converter will attempt to convert the same objects multiple times if they are found in the same Scenes multiple times. For instance

```
Scene
|___MyArmature
    |___Cube

Scene.001
|____MyArmature
    |____Lamp
```

Will attempt to convert MyArmature's animations twice, which could be catastrophic.

# Solutions
1. Add a "CONVERTED" game property to converted objects
Pros: Very simple, user won't see unless they look for it (and they can if needed!)
Cons: The game properties are a "2.49 thing", and we were going for a "no new game properties", new stateful data to manage.
2. Each function returns a list of converted objects, which are skipped
Pros: Also simple, prevents conversion functions from having to worry about state
Cons: Converter can only ever be run once, since this list is in memory.
3. Store converted objects somewhere in converter, such as a CollectionProp of names
Pros: Simple, prevents conversion functions from worrying about state. Allows converter to be run again. No use of game properties, but still saves to disk
Cons: What if an object's name changes? Then it gets reconverted as fresh data.

**Do we want the user to be able to re-run the converter?** Do we want developers to be able to re-run the converter?

**Comment by tngreene** [comment]
For now, I think we'll choose the "Run time only" option until we can decide if there is any use or meaning to re-running the converter. We'll obviously never have the ability to roll back the changes (too complicated), and a partially converted file could be impossible to really continue on again.

I think we're going to assume runtime only, one run only.

**Comment by tngreene** [comment]
Perhaps I'm no longer understanding my own bug, but, closed with 7be214c8ea983f
